Src/CZICmd/consoleio.h

```diff
@@ -4,6 +4,9 @@
 
 #pragma once
 
+#include <string>
+#include <memory>
```

Comment:
Can't immediatly deduce why these are neccessary. I'm sure the are, just wanted to check :)

Reply:
well, because std::string and std::shared_ptr appear further down in this header, and at least I'm striving for that every header is self-contained.